front/src/common/ReleaseInformations/LicenseAttributions.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { licenses, License } from './consts';
+
+const LicenseAttributions = () => {
+  const { t } = useTranslation();
+
+  const [packages, setPackages] = useState<License[]>([]);
+
+  useEffect(() => {
+    const tempArray: License[] = [];
+
+    Object.keys(licenses).forEach((key) => {
+      tempArray.push({ package: { title: key, data: licenses[key] } });
+    });
+
+    setPackages(tempArray);
+  }, []);
+
+  const attributions = packages.map((item) => (
+    <div className="d-flex flex-column align-items-start" key={item.package.data.name}>
+      <div className="d-flex">
+        <h3 className="mr-1">{item.package.data.name}</h3>{' '}
+        <span className="informations-version">{`(${item.package.data.version})`}</span>
+      </div>
+      <span className="pl-4 mb-4">
+        {item.package.data.copyright ||
+          item.package.data.publisher ||
+          `${item.package.data.name} authors`}
+      </span>
+    </div>
+  ));
```

Comment:
```suggestion
const attributions = Object.values(licenses).map(({ name, version, copyright, publisher }) => (
    <div className="d-flex flex-column align-items-start" key={name}>
      <div className="d-flex">
        <h3 className="mr-1">{name}</h3>{' '}
        <span className="informations-version">{`(${version})`}</span>
      </div>
      <span className="pl-4 mb-4">{copyright || publisher || `${name} authors`}</span>
    </div>
  ));
```
This is enough :) you don't need to make things complicated haha

Reply:
You can also remove the License type which is not used anymore !

Reply:
done